docker/drama-free-django/_build.sh

```diff
@@ -62,13 +49,14 @@ no-drama build "${build_args[@]}"
 echo "{}" > ./dfd_env.json
 
 # This is used by DFD to set Django's settings.STATIC_ROOT.
-echo '{"static_out": "../../../static"}' > ./dfd_paths.json
+# Q: Why do we need to override the default?
+# echo '{"static_out": "../static"}' > ./dfd_paths.json
```

Comment:
This is needed because [the default drama-free-django behavior](https://github.com/cfpb/drama-free-django/blob/113003e842ab93059b8f15de2828d24051f7e009/no_drama/build_skel/lib/dfd.py#L17) is to set its `static_out` variable to `'static'`. This is used to set Django [`STATIC_ROOT`](https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-STATIC_ROOT), which specifies where collectstatic puts its files. The default behavior would have this command collect staticfiles to a `static` relative path within the DFD deploy.

We instead want these files to go to `'../../../static/'`, in practice going from, say, `/srv/cfgov/versions/20190712095508/current` to `/srv/cfgov/static`.

@rosskarchner's open PR #5133 currently includes [a change](https://github.com/cfpb/cfgov-refresh/pull/5133/files#diff-ac27c5919c46548eada7a7388e726c59L244) that would set the default static root to `/srv/cfgov/static` as desired, but that'll only work once https://github.com/cfpb/drama-free-django/pull/25 or something like it removes the DFD behavior that overwrites the static root at runtime.

Reply:
Makes sense.  I'll change it back.